Tests/ProtocolBufferConversionTests.swift

```diff
@@ -448,6 +448,9 @@ final class ProtocolBufferConversionTests: XCTestCase {
 
   func testConvertPaymentTransactionAllCommonFieldsSet() {
     // GIVEN a Transaction protocol buffer with all common fields set.
+    let hash = Data([2, 4, 6])
+    let timestamp: UInt32 = 0
```

Comment:
Similar to the comment on Java, I think this should be UInt64 since adding 946,684,800 + <arbitrary int value> could overflow. 

The (documentation also notes this](https://xrpl.org/basic-data-types.html#specifying-time):
> Don't convert Ripple Epoch times to UNIX Epoch times in 32-bit variables: this could lead to integer overflows.

Reply:
So this particular value is just being fed to construct the `Date` protobuf... which requires a UInt32.  The `timestamp` field in the actual `XRPTransaction` object is indeed a `UInt64`, and the `UInt32` coming out of the protobuf gets converted to a `UInt64` before conversion, so I think we're good to go here.